src/P2/main.c

```diff
@@ -1,5 +1,6 @@
 #include "common.h"
 #include <game.h>
+#include "prog.h"
```

Comment:
I've been using `<>` for all includes except `common.h` just to keep things consistent